rpm/systemd/diamond.service

```diff
@@ -2,8 +2,9 @@
 Description=diamond - A system statistics collector for graphite
 
 [Service]
-ExecStart=/usr/bin/python /usr/bin/diamond --log-stdout --foreground
-Restart=on-abort
+ExecStart=/usr/bin/python /usr/bin/diamond -p /var/run/diamond.pid
```

Comment:
> we should put a comment on the conf file and systemd unit

works for me

> can't remove PIDFile option because systemd needs it 

Correct, i was reading about that last night (didn't test it though)

> another type of unit like "simple" 

From my reading `forking` looks like it would be more correct in this case